scripts/server-start.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# wrapper script for wait-for-it.sh and pubsweet server
+
+pid=0
+
+sigterm_handler () {
+  if [ $pid -ne 0 ]; then
+    kill -SIGTERM "$pid"
+    wait "$pid"
+  fi
+  exit 143;
+}
+
+trap 'kill ${!}; sigterm_handler' SIGTERM
+
+./scripts/wait-for-it.sh postgres:5432 -s -t 40
```

Comment:
the hardcoded `postgres` stops this from being used in production/staging, should use `PGHOST` and `PGPORT` (making sure they are both specified)

Reply:
I don't think i understand enough about the staging/production environment. Maybe we can chat on slack?

Reply:
Yeah ... let's talk f2f. IMHO ... we have far too many ways / environments in which the app is started - and the different ways are spread all over the place. At the very least we should document each of these. There are some docs in `docs/deploying` but certainly not enough to fully understand what is going on.